pytheus/metrics.py

```diff
@@ -153,6 +180,12 @@ def count_exceptions(
             self.inc()
             raise
 
+    def collect(self) -> Sample:
+        # TODO: probably need a way to add default metric creation labels
+        # ideally on Metric class initialization
+        # while being careful of not messing up the tracking logic
+        return Sample('_total', self._labels, self._value.get())
```

Comment:
I assume this suffix is some Prometheus convention, like in my 1st question

Reply:
yap! 😁